### PR TITLE
Cmr 8924 deploy grid schema to cmr

### PR DESCRIPTION
**Pull request by sxu123**

### CODE REVIEW

File: schemas/resources/schemas/data-quality-summary/v1.0.0/search.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ These parameters will match fields within a Data Quality Summary. They are case
 * provider - options: pattern, ignore\_case
 * native\_id - options: pattern, ignore\_case
 * concept\_id
-* id (legacy identifier)
+* id
```
Comment (eereiter) [inline schemas/resources/schemas/data-quality-summary/v1.0.0/search.md:29]: Please undo this change.  The id is the legacy services identifier. It isn't a random id.

File: schemas/resources/schemas/grid/v0.0.1/metadata.json [hunk 2]
```diff
@@ -11,53 +11,64 @@
     "GridDefinition": {
         "CoordinateReferenceSystemID": {
             "Type": "EPSG",
-			      "Code": "EPSG:4326",
+			"Code": "EPSG:4326",
```
Comment (eereiter) [inline schemas/resources/schemas/grid/v0.0.1/metadata.json:14]: Spacing is off here.  Code should be under type.

**Review by jceaser — APPROVED** [review]
approved if you follow up with erich's request.

File: schemas/resources/schemas/data-quality-summary/v1.0.0/search.md (hunk 1 above)
Comment (sxu123) [reply schemas/resources/schemas/data-quality-summary/v1.0.0/search.md:29]: They specifically change it to just id.

Comment (sxu123) [reply schemas/resources/schemas/data-quality-summary/v1.0.0/search.md:29]: in all the files.

File: schemas/resources/schemas/grid/v0.0.1/metadata.json (hunk 2 above)
Comment (sxu123) [reply schemas/resources/schemas/grid/v0.0.1/metadata.json]: fixed.

**Review by eereiter — APPROVED** [review]
